install_config/install/advanced_install.adoc

```diff
@@ -1181,6 +1181,139 @@ xref:../../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#i
 openshift_hosted_logging_storage_kind=dynamic
 ----
 
+[[enabling-service-catalog]]
+=== Enabling the Service Catalog
+
+[NOTE]
+====
+The service catalog is currently a Technology Preview feature.
+====
+
+Enabling the
+xref:../../architecture/service_catalog/index.adoc#architecture-additional-concepts-service-catalog[service catalog] allows service brokers to be registered with the catalog. The web
+console is also configured to enable an updated landing page for browsing the
+catalog.
```

Comment:
Without fully understanding what the service catalog is, I feel like "Enabling the service catalog" and "allows service brokers to be registered with the catalog" mean the same thing. Maybe:

"Enable the xref:../../architecture/service_catalog/index.adoc#architecture-additional-concepts-service-catalog[service catalog] so that users can <blah>. Once enabled, the landing page for the web console features a section for browsing the catalog."